Source/WebCore/rendering/svg/legacy/LegacyRenderSVGShape.cpp

```diff
@@ -264,9 +264,17 @@ void LegacyRenderSVGShape::fillStrokeMarkers(PaintInfo& childPaintInfo)
 
 void LegacyRenderSVGShape::paint(PaintInfo& paintInfo, const LayoutPoint&)
 {
-    if (paintInfo.context().paintingDisabled() || paintInfo.phase != PaintPhase::Foreground
-        || style().visibility() == Visibility::Hidden || isEmpty())
+    if (style().visibility() == Visibility::Hidden || isEmpty())
         return;
+
+    if (paintInfo.phase == PaintPhase::EventRegion) {
+        paintInfo.eventRegionContext()->unite(FloatRoundedRect(m_fillBoundingBox), *this, style(), false);
```

Comment:
Using the fillBoundingBox here to match `svgElement.toClipPath()`, don't know if we can easily get a Path for the "shaped including stroke"